internal/render/container.go

```diff
@@ -101,22 +101,22 @@ func (c Container) Render(o interface{}, name string, r *model1.Row) error {
 		return fmt.Errorf("expected ContainerRes, but got %T", o)
 	}
 
-	cur, res := gatherMetrics(co.Container, co.MX)
+	cur, res := gatherMetrics(co.Container(), co.MX)
```

Comment:
Making co.Container lazy eval is a killer here as we have to keep recomputing the container we're dealing with which is pretty much static.